core/extensions.rs

```diff
@@ -0,0 +1,136 @@
+use std::cell::RefCell;
+use std::rc::Rc;
+
+use crate::error::AnyError;
+use crate::{OpFn, OpId, OpState};
+
+pub type SourcePair = (&'static str, &'static str);
+pub type OpPair = (&'static str, Box<OpFn>);
+pub type RcOpRegistrar = Rc<RefCell<dyn OpRegistrar>>;
```

Comment:
I think I already mentioned it but IMO such alias adds even more indirection than using `Rc<RefCell<>>` directly

Reply:
Dropped the alias